lib/ast/node.rb

```diff
@@ -227,6 +227,21 @@ def inspect(indent=0)
     def to_ast
       self
     end
+    
+    # Produces a machine-readable s-expression.
+    #
+    # @return [Array]
+    def to_json
```

Comment:
Shouldn't `to_json` method return JSON?

Reply:
Good question. I was mistaken in community convention, looking at https://apidock.com/rails/Hash/to_json (I don't write a ton of ruby).